package.xml

```diff
@@ -0,0 +1,106 @@
+limitations under the License.
+-->
+<package version="2.0" 
+	xmlns="http://pear.php.net/dtd/package-2.0" 
+	xmlns:tasks="http://pear.php.net/dtd/tasks-1.0" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0 http://pear.php.net/dtd/tasks-1.0.xsd http://pear.php.net/dtd/package-2.0 http://pear.php.net/dtd/package-2.0.xsd">
+	<name>skywalking_agent</name>
+	<channel>pecl.php.net</channel>
+	<summary>Apache SkyWalking PHP Agent.</summary>
+	<description>The PHP Agent for Apache SkyWalking, which provides the native tracing abilities for PHP project.</description>
+	<lead>
+		<name>jmjoy</name>
+		<user>jmjoy</user>
+		<email>jmjoy@apache.org</email>
+		<active>yes</active>
+	</lead>
+	<lead>
+		<name>Yanlong He</name>
+		<user>yanlong</user>
+		<email>yanlong@php.net</email>
+		<active>yes</active>
+	</lead>
+	<date>2022-09-04</date>
+	<!-- TODO -->
+	<version>
+		<release>0.1.0</release>
+		<api>0.1.0</api>
+	</version>
+	<stability>
+		<release>stable</release>
+		<api>stable</api>
+	</stability>
+	<license uri="http://www.apache.org/licenses/LICENSE-2.0.html">Apache-2.0</license>
+	<notes>TODO</notes>
+	<contents>
+		<dir name="/">
+			<file role="doc" name=".asf.yaml" />
+			<file role="doc" name=".licenserc.yaml" />
+			<file role="doc" name=".rustfmt.toml" />
+			<file role="doc" name="LICENSE" />
+			<file role="doc" name="NOTICE" />
+			<file role="doc" name="README.md" />
+			<file role="src" name="Cargo.lock" />
+			<file role="src" name="Cargo.toml" />
+			<file role="src" name="config.m4" />
+			<file role="src" name="docker-compose.yml" />
+			<file role="src" name="src/channel.rs" />
+			<file role="src" name="src/component.rs" />
+			<file role="src" name="src/context.rs" />
+			<file role="src" name="src/execute.rs" />
+			<file role="src" name="src/global.rs" />
+			<file role="src" name="src/lib.rs" />
+			<file role="src" name="src/module.rs" />
+			<file role="src" name="src/plugin/mod.rs" />
+			<file role="src" name="src/plugin/plugin_curl.rs" />
+			<file role="src" name="src/plugin/plugin_swoole.rs" />
+			<file role="src" name="src/request.rs" />
+			<file role="src" name="src/util.rs" />
+			<file role="src" name="src/worker.rs" />
+		</dir>
+	</contents>
+	<dependencies>
+		<required>
+			<php>
+				<min>7.1.0</min>
+			</php>
+			<pearinstaller>
+				<min>1.4.0</min>
+			</pearinstaller>
+			<os>
+				<name>windows</name>
+				<conflicts/>
+			</os>
+		</required>
+	</dependencies>
+	<providesextension>skywalking_agent</providesextension>
+	<extsrcrelease>
+		<configureoption default="no" name="enable-cargo-debug" prompt="enable cargo debug?" />
+	</extsrcrelease>
+	<changelog>
+		<release>
+			<version>
+				<release>0.1.0</release>
+				<api>0.1.0</api>
+			</version>
+			<stability>
+				<release>stable</release>
+				<api>stable</api>
+			</stability>
+			<notes>TODO</notes>
+		</release>
+	</changelog>
```

Comment:
Is `<changelog>` necessary? I found that swoole was not written, and this was a little cumbersome and superfluous.

Reply:
This is not required, PECL will record change log.

Reply:
I found swoole was not written, but it has:

Reply:
Maybe record from `notes` and `version` tag

Reply:
Delete it and try try the `pecl.php.net` will or will not has the `<changelog>`?

Reply:
Yes, Delete it